### ISSUE TITLE
Custom validators with name length restrictions

### ISSUE THREAD

**Issue by gkaimakas** [balderdashy/sails#2598]
I was trying to make some custom validators for my models the other day and I was trying this name for one validator: 

_isI18nObject

when I run some unit test, it appeared that all my models failed to do anything because the validation was failing on my custom validator. Since I had run some unit tests on the login behind the validator I was sure that the logic was fine and indeed when I changed the name to

_isI18nObj

everything worked great.

Is this some sort of requirement, "keep validators small etc", or a bug ?

**Comment by CWyrtzen** [comment]
Is this still an issue? Is there anything about it in waterline repos? @particlebanana ?

**Comment by sailsbot** [comment]
Thanks for posting, @gkaimakas.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
